site/templates/items/tagline.jet

```diff
@@ -9,85 +13,56 @@
       {{end}}
     {{end}}
 
-    {*{ Figure out how many dividers. }*}
-    {{dividers := 0}}
-    {{if genres && len(itemGenres) > 0}}
-      {{dividers = dividers + 1}}
-    {{end}}
-    {{if isset(.Runtime)}}
-      {{dividers = dividers + 1}}
-    {{end}}
-    {{if isset(.Items)}}
-      {{dividers = dividers + 1}}
-    {{end}}
-    {{if isset(.Episodes)}}
-      {{dividers = dividers + 1}}
-    {{end}}
-    {{if isset(.ReleaseDate) && .ReleaseDate.Year() > 1}}
-      {{dividers = dividers + 1}}
-    {{end}}
-    {*{ There should be one less dividers than tagline items. }*}
-    {{dividers = dividers - 1}}
-    
-    {{if classification}}
-      <s72-classification-label data-slug="{{.Slug}}" data-layout="tooltip"></s72-classification-label>
-    {{end}}
+    {{showDivider := false}}
 
     {{if isset(.Runtime)}}
-      {{yield taglineItem() content}}
+      {{yield item(class="runtime") content}}
         {{if .Runtime > 60}}
-          {{.Runtime.Hours()}}{{i18n("runtime_hours")}} {{.Runtime.Minutes()}}{{i18n("runtime_minutes")}}
-        {{else}}
-          {{.Runtime}}{{i18n("runtime_minutes")}}
+          {{.Runtime.Hours()}}{{i18n("runtime_hours")}}
         {{end}}
+        {{.Runtime.Minutes()}}{{i18n("runtime_minutes")}}
       {{end}}
-      {{yield taglineDivider(dividers=dividers)}}
-      {{dividers = dividers - 1}}
+      {{showDivider = true}}
     {{end}}
 
     {{if isset(.Items)}}
-      {{yield taglineItem() content}}
+      {{yield item(class="items") content}}
         {{i18n("bundle_items_all_films", len(.Items))}}
       {{end}}
-      {{yield taglineDivider(dividers=dividers)}}
-      {{dividers = dividers - 1}}
+      {{showDivider = true}}
     {{end}}
 
     {{if isset(.Episodes)}}
-      {{yield taglineItem() content}}
+      {{yield item(class="episodes") content}}
         {{i18n("episode_count", len(.Episodes))}}
       {{end}}
-      {{yield taglineDivider(dividers=dividers)}}
-      {{dividers = dividers - 1}}
+      {{showDivider = true}}
     {{end}}
 
     {{if genres && len(itemGenres) > 0}}
       {{if genresLimit > -1 && len(itemGenres) > genresLimit}}
         {{itemGenres = itemGenres[:genresLimit]}}
       {{end}}
-      {{yield taglineItem() content}}
+      {{yield item(class="genres") content}}
         {{itemGenres}}
       {{end}}
-      {{yield taglineDivider(dividers=dividers)}}
-      {{dividers = dividers - 1}}
+      {{showDivider = true}}
     {{end}}
 
-    {{if isset(.ReleaseDate) && .ReleaseDate.Year() > 1}}
-      {{yield taglineItem() content}}
+    {{if isset(.ReleaseDate) && .ReleaseDate.Year() > 1888}}
```

Comment:
`1888` = first film ever 📽️

Reply:
No big deal either way but wondering if we want this to be any number for a clients "artistic" reasons